docs/getting-started.md

```diff
@@ -2,8 +2,16 @@
 
 ### 1. Install the kickstarter
 
+Using npm
+
+```sh
+$ npm install -g kickstarter-tools
+```
+
+Using yarn
+
 ```sh
-$ npm install -g {Kickstarter repo}
+$ yarn add -g kickstarter-tools
```

Comment:
I didn't find `-g` shortcut anywhere.

[Documentation](https://yarnpkg.com/fr/docs/cli/global) for global command